File: coco_csharp/CoCo/EP/M6/Finalized/Neg.cs
namespace EP.M6.Finalized {
    public class Neg : Exp, EP.M6.Neg<Exp> {
        private EP.Exp<Exp> inner;

        public Neg(EP.Exp<Exp> _inner) {
            this.inner = _inner;
        }

        public EP.Exp<Exp> getInner() { return this.inner; }
        public void setInner(EP.Exp<Exp> _inner) { this.inner = _inner; }
    }
}

File: trivially/src/main/java/trivially/ep/m5/finalized/Neg.java
package trivially.ep.m5.finalized;

import trivially.ep.m5.Exp;

public class Neg implements trivially.ep.m5.Neg {

    protected Exp inner;

    public Exp getInner() {
        return this.inner;
    }

    // Warning: Unsafe casts
    public void setInner(trivially.ep.m4.Exp inner) { this.inner = (Exp) inner; }

    public Neg(Exp inner) {
        this.inner = inner;
    }
}
